examples/keyfiguresfinland.py

```diff
@@ -0,0 +1,131 @@
+# -*- coding: utf-8 -*-
+import pandas as pd
+from dash import Dash, dcc, html, Input, Output
+import dash_bootstrap_components as dbc
+import plotly.express as px
+import json
+
+# Data provided by Statistics Finland.
+# Original data source: https://pxdata.stat.fi/PxWeb/pxweb/en/Kuntien_avainluvut/Kuntien_avainluvut__2021/kuntien_avainluvut_2021_viimeisin.px/
+df = pd.read_csv('assets/key_figures_regions.csv', encoding = 'latin-1').rename(columns ={'Region 2021':'Region'}).set_index('Region')
```

Comment:
Why `latin-1` encoding?

Reply:
Well I've noticed that these Statistics Finland's files require that. Might be an old habit though since I tend to work with files that have nordic alphabet :-)

Reply:
This actually does also have nordic alphabet since there are ä, ö, and å in region names